infra/base-images/base-runner/test_all.py

```diff
@@ -80,6 +80,11 @@ def find_fuzz_targets(directory):
       continue
     if filename.startswith('jazzer_'):
       continue
+    if os.getenv('FUZZING_ENGINE') == 'centipede':
+      if os.getenv('SANITIZER') in filename and os.path.isdir(path):
+        fuzz_targets.extend(find_fuzz_targets(path))
```

Comment:
what's this needed for? finding sanitizer binaries? Do we need this? Or should we just be returning the non-sanitized list?

Reply:
Yes, it finds the sanitized binaries only.
`bad_build_check` assumes all the binaries in the `fuzz_targets` list are sanitized when we specify [`SANITIZER=address`](https://github.com/google/oss-fuzz/blob/2d5d3eba7d5cc09a6b0d7e3961c4c27b127574c1/infra/base-images/base-runner/bad_build_check#L329): It [verifies if the binary is built with `ASan`](https://github.com/google/oss-fuzz/blob/2d5d3eba7d5cc09a6b0d7e3961c4c27b127574c1/infra/base-images/base-runner/bad_build_check#L168) and fail otherwise.
I excluded the unsanitized binary so that it won't fail the test, and used sanitized binaries to make use of `bad_build_check`'s feature to check if the sanitized binary is built with the correct sanitizer.